src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -207,6 +216,16 @@ export default {
 		attachmentFolderFreeSpace() {
 			return this.$store.getters.getAttachmentFolderFreeSpace()
 		},
+
+		currentConversationIsJoined() {
+			return this.$store.getters.currentConversationIsJoined
+		},
+	},
+
+	watch: {
+		disabled(newValue) {
+			this.$refs.uploadMenu.$refs.menuButton.disabled = newValue
```

Comment:
are we missing a "disabled" attribute in uploadMenu ?

let's add a "FIXME" here with a link to a ticket to do so.
otherwise this approach looks really ugly

Reply:
So actually the disable is missing on the Actions component.
Please upstream it to the vue lib directly.